assign-doi-proxy-function/src/test/java/no/unit/nva/datacite/DataCiteMdsConnectionTest.java

```diff
@@ -0,0 +1,148 @@
+package no.unit.nva.datacite;
+
+import nva.commons.utils.IoUtils;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.net.http.HttpResponse;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+public class DataCiteMdsConnectionTest {
```

Comment:
`assign-doi-proxy` function is not included in the modules `settings-gradle-file` and this code is excluded from compilation or testing. Include the module in the project.

Reply:
If this code is not used,delete it.